plugins/processors/k8sdecorator/stores/utils_test.go

```diff
@@ -14,7 +16,14 @@ func TestUtils_parseDeploymentFromReplicaSet(t *testing.T) {
 }
 
 func TestUtils_parseCronJobFromJob(t *testing.T) {
-	assert.Equal(t, "", parseCronJobFromJob("hello-123"))
-	assert.Equal(t, "hello", parseCronJobFromJob("hello-1234567890"))
-	assert.Equal(t, "", parseCronJobFromJob("hello-123456789a"))
+	//For CronJobControllV2 which is after K8s v1.21
```

Comment:
I think iterating over a set of inputs/outputs would make this test clearer. See https://github.com/aws/amazon-cloudwatch-agent/blob/master/cfg/agentinfo/info_test.go#L104 for an example.

Reply:
Yes. Changed to use table test driven and just committed. Takes long time to cover the coverage test case and goes beyond it.